packages/wasm/package.json

```diff
@@ -1,27 +1,32 @@
 {
   "name": "@penumbra-zone/wasm",
   "version": "3.0.0",
-  "private": true,
   "license": "MIT",
   "type": "module",
   "scripts": {
     "clean": "rm -rfv dist wasm ; cargo clean --manifest-path ./crate/Cargo.toml",
     "dev": "cargo watch -C ./crate --postpone -- pnpm turbo run compile",
     "compile": "cd crate ; wasm-pack build --no-pack --target bundler --out-name index --out-dir ../wasm",
-    "build": "tsc",
+    "prepack": "touch ./wasm/.npmignore",
```

Comment:
What does this do/why is it required?

Reply:
this prevents ignoring the compiled wasm when packaging by overriding the wildcard gitignore

Reply:
prepack is called during publishing